.github/workflows/checks.yml

```diff
@@ -11,7 +11,7 @@ jobs:
         - python-version: "3.11"
           env:
             TOXENV: pylint
-        - python-version: 3.7
+        - python-version: 3.8
```

Comment:
Needed because `types-lxml` is not available for 3.7.

3.7 EOL is in 4 months so I don't think this is a big problem.